### PR TITLE
Regex correction & Formatting run

### PR DESCRIPTION
**Pull request by IllustrisJack**
+ manual formatting fixes for @Flatfingers

### CODE REVIEW

File: script/GameObjects/Elements/Components/Capacitor.lua
Comment (Flatfingers) [inline script/GameObjects/Elements/Components/Capacitor.lua:1]: The spaces were part of the formatting for readability on these two comments.

Comment (IllustrisJack) [reply script/GameObjects/Elements/Components/Capacitor.lua:1]: ok will correct

File: script/GameObjects/Elements/Components/Shield.lua
Comment (Flatfingers) [inline script/GameObjects/Elements/Components/Shield.lua:1]: Here as well.

File: script/GameObjects/Elements/Economy/Content.lua
Comment (Flatfingers) [inline script/GameObjects/Elements/Economy/Content.lua:1]: The spaces here were being used on purpose to make the words line up with the data values below them.

Comment (IllustrisJack) [reply script/GameObjects/Elements/Economy/Content.lua:1]: I figured, hence why i replaced it with

Item(NAME, UNIT, MASS, ENERGY-DENSITY, DISTRIBUTION %)

Showing the object structure instead. The previous formatting was: `-- NAME UNIT MASS ENERGY-DENSITY DISTRIBUTION %`, from which i can see does not line up with the data values.

Comment (IllustrisJack) [reply script/GameObjects/Elements/Economy/Content.lua:1]: (i suppose you are talking about the whitespaces before the formatter ran over it the last time)

File: script/Systems/Controls/Control.lua
Comment (Flatfingers) [inline script/Systems/Controls/Control.lua:1]: Good changes.

**Review by Flatfingers — APPROVED** [review]

**Pull request merged**